open_event/api/login.py

```diff
@@ -0,0 +1,31 @@
+import json
+from flask.ext.restplus import Resource, Namespace, fields
+from custom_fields import EmailField
+from helpers import _error_abort
+from flask_jwt import JWTError
+
+api = Namespace('login', description='Login')
+
+LOGIN = api.model('Login', {
+    'email': EmailField(required=True),
+    'password': fields.String(required=True)
+})
+
+TOKEN = api.model('Token', {
+    'access_token': fields.String
+})
+
+
+@api.route('')
+class Login(Resource):
+    @api.doc('get_token')
+    @api.expect(LOGIN, validate=True)
+    @api.marshal_with(TOKEN)
+    @api.response(401, 'Authentication Failed')
+    def post(self):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/triplez/open-event-orga-server/file/2716743256/issues/source?bid=3233864&fileBranchId=3345807#l25)